compiler/damlc/tests/daml-test-files/SemanticsEvalOrder.daml

```diff
@@ -530,51 +530,44 @@ evUpdExercNonConsumErr = scenario do
     abort "EvUpdExercNonConsumErr failed"
 
 
--- TODO https://github.com/digital-asset/daml/issues/8020
```

Comment:
There's also the same issue in the choice observer field. The test is in a different file, `SemanticsEvalOrderSince1_11.daml`. Please uncomment the tests there as well :)

Reply:
Good catch 0ad6148062eecf773607c590d578005f495fbe53